apps/omg_db/lib/omg_db/leveldb_server.ex

```diff
@@ -64,7 +64,26 @@ defmodule OMG.DB.LevelDBServer do
     end
   end
 
-  def handle_call({:multi_update, db_updates}, _from, state) do
+  def handle_call({:multi_update, db_updates}, _from, state), do: multi_update(db_updates, state)
```

Comment:
I think we have two options here:

1/ add a `OMG.DBServer` behavior which implements this, which has callbacks like `multi_update`, which calls `GenServer.call(__MODULE__...)` and is in turn called by `OMG.DB`. Then you'll decorate the callback implementations in `LevelDBServer` - preferred option but more work
2/ rename to `do_multi_update` and alike and keep the `do_` functions decorated private - lazy option, just to be consistent

We can also consider
3/ anticipate doing (1/) in a separate PR and keep the current name, but leave a `TODO behind`, copying (1/) into the body of the TODO